bool[] obs
---
nav_msgs/OccupancyGrid map
bool result
